rts/motoko-rts/src/region.rs

```diff
@@ -591,12 +571,32 @@ pub(crate) unsafe fn region_migration_from_no_stable_memory<M: Memory>(mem: &mut
     // Region 0 -- classic API for stable memory, as a dedicated region.
     REGION_0 = region_new(mem);
 
-    assert!((*REGION_0.as_region()).id == 0);
+    assert!((*REGION_0.as_region()).id_lower == 0);
+    assert!((*REGION_0.as_region()).id_upper == 0);
 
     // Regions 1 through LAST_RESERVED_REGION_ID, reserved for future use by future Motoko compiler-RTS features.
     region_reserve_id_span(mem, Some(RegionId(1)), RegionId(LAST_RESERVED_REGION_ID));
 }
 
+pub fn block_page_count(rank: u16, block_count: u32, page_count: u32) -> u8 {
+    use meta_data::size::PAGES_IN_BLOCK;
```

Comment:
Idea (not sure whether really useful): A static check that `PAGES_IN_BLOCK` fits into u8 could prevent unwanted changes of `max::PAGES_IN_BLOCK` and also explain that the downcasts are safe.

Reply:
how do I do a static assert in Rust - do we have other examples in the rts you can point me at?

Reply:
I believe an assert statement would be optimized away if it can be statically evaluated.
Explicit static assertions can be written like:
`const _: () = assert!(size_of::<usize>() == size_of::<u32>());`

Reply:
Thanks! Done